mono/metadata/class.c

```diff
@@ -199,18 +199,17 @@ mono_class_from_typeref (MonoImage *image, guint32 type_token)
  * not be loaded with the \p error value filled with the information about the
  * error.
  */
-MonoClass *
+MONO_API MonoClass *
```

Comment:
If the declaration in the `.h` file is marked as `MONO_API` then there is no need to do it in the `.c` file too.